libvast/src/detail/line_range.cpp

```diff
@@ -20,7 +20,6 @@ namespace vast {
 namespace detail {
 
 line_range::line_range(std::istream& input) : input_{input} {
-  next(); // prime the pump
```

Comment:
Have you checked that all line-based formats now issue their first call to `next` lazily?

Reply:
I modified `zeek`, `json`, `csv` and `syslog`. I just found one more user of `line_range`: `parser_reader`, but it isn't used...